src/main/java/seedu/address/ui/QuizWindow.java

```diff
@@ -123,15 +122,21 @@ void fillInnerParts() {
         resultDisplay = new ResultDisplay();
         resultDisplayPlaceholder.getChildren().add(resultDisplay.getRoot());
 
-        // TODO: Change this back
-        //StatusBarFooter statusBarFooter = new StatusBarFooter(logic.getAddressBookFilePath());
-        StatusBarFooter statusBarFooter = new StatusBarFooter(Path.of("This is the quiz window"));
-        statusbarPlaceholder.getChildren().add(statusBarFooter.getRoot());
+        updateStatusBar();
 
         CommandBox commandBox = new CommandBox(this::executeCommand);
         commandBoxPlaceholder.getChildren().add(commandBox.getRoot());
     }
 
+    /**
+     * Updates the status bar with the current question number.
+     */
+    private void updateStatusBar() {
+        StatusBarFooter statusBarFooter = new StatusBarFooter(String.format("Question %d of %d",
+                quizManager.getCurrentIndex() + 1, quizManager.getTotalQuestions()));
+        statusbarPlaceholder.getChildren().add(statusBarFooter.getRoot());
```

Comment:
For this, is it better to keep it in variables for better readability?
e.g.
```java
currentQuestionNumber = quizManager.getCurrentIndex() + 1
totalQuestions = quizManager.getTotalQuestions()
StatusBarFooter statusBarFooter = new StatusBarFooter(String.format("Question %d of %d",
                currentQuestionNumber , totalQuestions))
```

Reply:
Updated accordingly